lib/rpmsg/rpmsg.c

```diff
@@ -249,9 +249,18 @@ static void rpmsg_unregister_endpoint(struct rpmsg_endpoint *ept)
 	metal_mutex_release(&rdev->lock);
 }
 
-void rpmsg_register_endpoint(struct rpmsg_device *rdev,
-			     struct rpmsg_endpoint *ept)
+void rpmsg_init_ept(struct rpmsg_endpoint *ept,
```

Comment:
Please could you propose this as a separate patch?
What would be the benefit of merging rpmsg_register_endpoint and rpmsg_initialize_ept? Do you see an improvement in the code size?

Reply:
Done. should be same as before